modules/dnn/test/test_tf_importer.cpp

```diff
@@ -1059,7 +1059,11 @@ TEST_P(Test_TensorFlow_layers, keras_mobilenet_head)
 TEST_P(Test_TensorFlow_layers, resize_bilinear)
 {
     runTensorFlowNet("resize_bilinear");
+    runTensorFlowNet("resize_bilinear",        false, 0.0, 0.0, false, "_acTrue_hpixcFalse");
+    runTensorFlowNet("resize_bilinear",        false, 0.0, 0.0, false, "_acFalse_hpixcTrue");
     runTensorFlowNet("resize_bilinear_factor");
+    runTensorFlowNet("resize_bilinear_factor", false, 0.0, 0.0, false, "_acFalse_hpixcTrue");
+    runTensorFlowNet("resize_bilinear_factor", false, 0.0, 0.0, false, "_acTrue_hpixcFalse");
```

Comment:
Why not just name test files as resize_bilinear_factor_half_pixel?

Reply:
Corrected.

Reply:
I mean to remove `groupPrefix`

Reply:
`groupPrefix` was introduced in order to reuse the same input data for test groups, where the group is, for example, `resize_bilinear` and derived cases with different parameters. Thus, we can reduce extra input data generation in opencv-extra (also added appropriate changes in generation script).